### PR TITLE
Сhange env variables as in other fusion clients

### PR DESCRIPTION
**Pull request by DenDen039**
Changed naming of environment variables to match Terraform Provider and Ansible Collection

### CODE REVIEW

**Review by mikenelson-io — APPROVED** [review]
We must ensure that these are consistent across all SDKs we provide.

File: python/utils.py [hunk 1]
```diff
@@ -23,18 +29,21 @@ def get_fusion_config() -> fusion.Configuration:
...
-    config.private_key_file = os.environ["PRIV_KEY_FILE"]
+    if (ENV_VAR_FUSION_ISSUER_ID not in os.environ or ENV_VAR_FUSION_PRIVATE_KEY_FILE not in os.environ) and ENV_VAR_FUSION_ACCESS_TOKEN not in os.environ:
+        raise ValueError(
+            f"Environmental variables neither '{ENV_VAR_FUSION_ISSUER_ID}' and '{ENV_VAR_FUSION_PRIVATE_KEY_FILE} nor '{ENV_VAR_FUSION_ACCESS_TOKEN}' are set!"
+        )
+
+    if ENV_VAR_FUSION_ACCESS_TOKEN in os.environ:
+        config.access_token = os.environ[ENV_VAR_FUSION_ACCESS_TOKEN]
+    config.issuer_id = os.environ[ENV_VAR_FUSION_ISSUER_ID]
+    config.private_key_file = os.environ[ENV_VAR_FUSION_PRIVATE_KEY_FILE]
```
Comment (andreepyro) [inline python/utils.py:40]: Here `KeyError` can be raised. Please address this (add try/except or check whether the values are present).

File: python/utils.py [hunk 2]
```diff
@@ -23,18 +29,21 @@ def get_fusion_config() -> fusion.Configuration:
...
 
     # optional values
-    if "HOST_ENDPOINT" in os.environ:
-        config.host = os.environ["HOST_ENDPOINT"]
-    if "TOKEN_ENDPOINT" in os.environ:
-        config.token_endpoint = os.environ["TOKEN_ENDPOINT"]
+    if ENV_VAR_FUSION_API_HOST in os.environ:
+        config.host = os.environ[ENV_VAR_FUSION_API_HOST] + "/api/1.0"
```
Comment (andreepyro) [inline python/utils.py:44]: Are we sure to append `"/api/1.0"`? Do we have it in other SDKs too?

**Review by andreepyro — CHANGES_REQUESTED** [review]

File: python/utils.py (hunk 2 above)
Comment (dimleevik) [inline python/utils.py:44]: Maybe it is a better idea to use some library for concationation of two url parts, what if `os.environ[ENV_VAR_FUSION_API_HOST]` will have `/`symbol in the end? Using something like urllib?

**Review by dimleevik — CHANGES_REQUESTED** [review]

**Review by andreepyro — APPROVED** [review]
LGTM

**Review by dimleevik — APPROVED** [review]

**Pull request merged**